src/platform/device.gni

```diff
@@ -55,10 +55,10 @@ declare_args() {
 
   # Enable wifi support.
   chip_enable_wifi =
-      chip_device_platform == "linux" || chip_device_platform == "esp32" ||
-      chip_device_platform == "mbed" || chip_device_platform == "tizen" ||
-      chip_device_platform == "android" || chip_device_platform == "ameba" ||
-      chip_device_platform == "webos" || chip_device_platform == "bl602"
+      chip_device_platform == "linux" || chip_device_platform == "mbed" ||
```

Comment:
Is this change needed? the enable_wifi seems to be a default, so "on" or "off" for a platform that supports both does not seem to be as relevant. Not having this change seems to result in a smaller delta.

Reply:
removed this change.